UnitystationLauncher/Models/Server.cs

```diff
@@ -34,9 +34,32 @@ public Server(string forkName, int buildVersion, string serverIp, int serverPort
             RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? LinuxDownload :
             throw new InvalidOperationException("Failed to detect OS");
 
+        public bool HasTrustedUrlSource
+        {
+            get
+            {
+                const string trustedHost = "unitystationfile.b-cdn.net";
+                var urls = new[] { WinDownload, OsxDownload, LinuxDownload };
+                foreach (var url in urls)
+                {
+                    if (string.IsNullOrEmpty(url))
+                    {
+                        return false;
+                    }
+                    var uri = new Uri(url);
+                    if (uri.Scheme != "https" && uri.Host != trustedHost)
+                    {
+                        return false;
+                    }
+                }
+                return true;
+            }
+        }
+
         public string InstallationName => ForkName + BuildVersion;
 
-        public string Description => $"BuildVersion: {BuildVersion} - Map: {CurrentMap} - Gamemode: {GameMode} - Time: {InGameTime}";
+        public string Description =>
+            $"BuildVersion: {BuildVersion} - Map: {CurrentMap} - Gamemode: {GameMode} - Time: {InGameTime}";
```

Comment:
I thought this was just `dotnet-format` that changed this but it doesn't seem to be the case.
Since this is unrelated to the issue it should be reverted.

Reply:
done